parent/pom.xml

```diff
@@ -1335,6 +1336,11 @@
       </dependency>
       <dependency>
         <groupId>org.apache.camel</groupId>
+        <artifactId>camel-iec60870</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.camel</groupId>
```

Comment:
I though the starters get added in `camel.git/platforms/spring-boot/components-starter/pom.xml`, there it is present.